.travis.yml

```diff
@@ -37,6 +37,7 @@ jobs:
     - stage: "Test on amd64"
       name: "build, smallbuild, crossbuild"
       arch: amd64
+      dist: trusty
```

Comment:
There's some crossbuild error with 16.04, so using 14.04 for workaround here.
/cc @kevin-wangzefeng @chendave @kadisi

Reply:
I'm fine to get the fix in with this restriction, please create a follow-up issue to track this.

Reply:
https://github.com/kubeedge/kubeedge/pull/1183  There is already a WIP PR to solve this issue, but it‘s a little complicated that relate to the vm image of travis.

Reply:
OK